lib/env.js

```diff
@@ -21,8 +22,8 @@ exports = module.exports = createEnv;
  */
 
 function Env (envfile) {
-  this.envfile = path.join(process.cwd(), envfile || 'env.json')
-  this.evars = (path.existsSync(this.envfile)) ? require(this.envfile) : {}
+  this.envfile = envfile || path.resolve(process.cwd() + '/env.json');
+  this.evars = (fs.statSync(this.envfile)) && require(this.envfile)
```

Comment:
We use `fs.statSync` because if the `env.json` file does not exist most apps probably wont work to the error (with full stack trace) is appropriate. what do you think?